iOS-sample-app/Showcases/Privacy/PrivacyViewController.swift

```diff
@@ -0,0 +1,114 @@
+import UIKit
+import ContentSquare
+
+// This class showcases how to use privacy-related APIs
+class PrivacyViewController: UIViewController {
+
+    private let commandTitle: UILabel = UILabel()
+    private let commandMessage: UILabel = UILabel()
+
+    override func loadView()
```

Comment:
Why not create these in the storyboard ? I'd argue it adds lines here that might confuse some (albeit most should be fine with it of course) and, more importantly, it makes this class less focused on showcasing how to use the SDK.

Reply:
Yes, I will change these for storyboard, as I did for the other stories